packages/@pollyjs/adapter-fetch/CHANGELOG.md

```diff
@@ -7,115 +7,80 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 **Note:** Version bump only for package @pollyjs/adapter-fetch
```

Comment:
All `CHANGELOG.md` files should be ignored from linting as they are auto generated.

Reply:
Done.